testsupport/tiers/checks.go

```diff
@@ -300,7 +284,55 @@ func rbacEditRoleBinding() namespaceObjectsCheck {
 	}
 }
 
+func crtadminViewRoleBinding() namespaceObjectsCheck {
+	return func(t *testing.T, ns *v1.Namespace, memberAwait *wait.MemberAwaitility, userName string) {
```

Comment:
not a blocker here, but rather a general thought: I wonder if it's better to use Go API or external YAML file to describe the expectation. WDYT?

Reply:
what do you mean by Go API?

I'm fine with using it as it is, but also open to any discussion/proposals for improvement - feel free to add it to the next sync call agenda.

Reply:
as for the external file - I guess that it's better to use just go types & functions - it makes the objects/resources configurable for other tiers that just contain different numbers/values